nodejs/.gitkeep

```diff
@@ -0,0 +1 @@
+bin
```

Comment:
The `.gitkeep` file doesn't do anything but placeholder to keep the directory structure. If you want to include that `bin` directory, please use `.gitignore`.

You can safely remove this `.gitkeep` file.